Makefile

```diff
@@ -29,6 +29,10 @@ K3D_CLUSTER_NAME      ?= k3s-default
 KUBE_NAMESPACE        ?= argo
 MANAGED_NAMESPACE     ?= $(KUBE_NAMESPACE)
 
+E2E_TIMEOUT           ?= 1m
```

Comment:
Individual timeout? A bit confusing with suite timeout.

Reply:
This variable represents the timeouts for waiting on workflows, pods etc. We could change this to a different name ie. `E2E_WAIT` or `E2E_WAIT_TIMEOUT`.

Reply:
Thanks. I don't have a preference on names as long as we document them. Could you add a comment above the declaration or document in one of the developer guides?

Reply:
Yes! Will add in the Makefile.